requirements.txt

```diff
@@ -1,3 +1,9 @@
+httpx>=0.25.0
 onepasswordconnectsdk==1.*
+
+# the following two libraries have a major version bump; we need to update our usage, at least for the slack provider
```

Comment:
Thanks for figuring out that it is finally time to give the Slack provider usage some attention. 🙏 Pretty good that we made it all this way since it was written for v1! I'll take a look at that PR next.